connect/docs/api.md

```diff
@@ -0,0 +1,17 @@
+# API
+
+- `ylem()`
+    - `ylem(Component)` -> returns an observer component (useful for making more efficient renders)
+    - `ylem(VM, Component)` -> return and enhanced component
```

Comment:
I'd use `ObserveObject` or something else in place of `VM`

Reply:
👍